### PR TITLE
View recent

### PR DESCRIPTION
**Pull request by daboxu**

### CODE REVIEW

File: onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 /**
  * The class for the Base Delta Collection Page.
  */
-public class BaseDeltaCollectionPage extends BaseCollectionPage<Item, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
+public class BaseDeltaCollectionPage extends BaseCollectionPage<DeltaResponse, IDeltaRequestBuilder> implements IBaseDeltaCollectionPage {
```
Comment (peternied) [inline onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java]: This is a breaking change, we need to adjust this so DeltaCollectionPage continues to return items, not DeltaResponses

**Comment by peternied** [comment]
:shipit:

File: onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java (hunk 1 above)
Comment (daboxu) [inline onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java]: probably we want to do it for a separated one as the model generator has been changed?

Comment (peternied) [inline onedrivesdk/src/main/java/com/onedrive/sdk/generated/BaseDeltaCollectionPage.java]: When the template generator was changed this output was changed, we need to update the generator not to change how Delta is handled.

**Comment by daboxu** [comment]
reverted delta related, going to change the templates for handling delta the right way

**Comment by peternied** [comment]
:shipit:

**Comment by daboxu** [comment]
fix for #71

**Pull request merged**